ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: Feature request
+labels: feature request
+assignees: BainanXia, danielolsen, jon-hagg, rouille
+
+---
+
+ # :rocket:
+
+- [ ] Is your feature request essential for your project?
+
+
+### Describe the workflow you want to enable (Required)
+A clear and concise description of what can be enhanced, e.g., "I wish I could do [...]"
+
+### Describe your proposed implementation
```

Comment:
Based on our discussion, I might emphasize that this is optional or context dependent (as opposed to something we would prefer but isn't strictly required).

Reply:
I will remove "(Required)" in line 15 and add to line 18 ",if familiar with code base":
### Describe your proposed implementation, if familiar with code base

Would that work?

Reply:
Hmm, I might leave it as just "optional" or "if applicable" - even if someone is familiar with the code I'd say it's fine (and in some cases, expected) to omit implementation.